### PR TITLE
tests: Add Depth/Stencil input attachment test

### PR DESCRIPTION
**Pull request by spencer-lunarg**
there was a `VUID-VkWriteDescriptorSet-descriptorType-07729` but when I wrote this test I hit `VUID-VkDescriptorImageInfo-imageView-01976` before getting to it. The `07729` was [removed](  https://gitlab.khronos.org/vulkan/vulkan/-/merge_requests/5815)

This splits `DSAspectBitsErrors` and creates a dedicated `InputAttachmentDepthStencilAspect` and `MultiplaneAspectBits`

### CODE REVIEW

File: tests/negative/ycbcr.cpp [hunk 1]
```diff
@@ -1489,3 +1489,96 @@ TEST_F(VkLayerTest, MultiplaneIncompatibleViewFormat) {
...
+        nullptr, VkImageCreateFlags{VK_IMAGE_CREATE_MUTABLE_FORMAT_BIT},  // need for multi-planar
+        VK_IMAGE_TYPE_2D, mp_format, VkExtent3D{64, 64, 1}, 1u, 1u, VK_SAMPLE_COUNT_1_BIT, VK_IMAGE_TILING_OPTIMAL,
+        VkImageUsageFlags{VK_IMAGE_USAGE_SAMPLED_BIT}, VK_SHARING_MODE_EXCLUSIVE, 0u, nullptr, VK_IMAGE_LAYOUT_UNDEFINED);
+    image_obj.init(&image_ci);
+    ASSERT_TRUE(image_obj.initialized());
+
+    auto vkCreateSamplerYcbcrConversionKHR = reinterpret_cast<PFN_vkCreateSamplerYcbcrConversionKHR>(
+        vk::GetDeviceProcAddr(m_device->handle(), "vkCreateSamplerYcbcrConversionKHR"));
+    auto vkDestroySamplerYcbcrConversionKHR = reinterpret_cast<PFN_vkDestroySamplerYcbcrConversionKHR>(
+        vk::GetDeviceProcAddr(m_device->handle(), "vkDestroySamplerYcbcrConversionKHR"));
```
Comment (arno-lunarg) [inline tests/negative/ycbcr.cpp:1536]:
```suggestion
    auto vkCreateSamplerYcbcrConversionKHR = GetDeviceProcAddr<PFN_vkCreateSamplerYcbcrConversionKHR>("vkCreateSamplerYcbcrConversionKHR");
    auto vkDestroySamplerYcbcrConversionKHR = GetDeviceProcAddr<PFN_vkDestroySamplerYcbcrConversionKHR>("vkDestroySamplerYcbcrConversionKHR");
```

File: tests/negative/ycbcr.cpp [hunk 2]
```diff
@@ -1489,3 +1489,96 @@ TEST_F(VkLayerTest, MultiplaneIncompatibleViewFormat) {
...
+    OneOffDescriptorSet descriptor_set(
+        m_device, {
+                      {0, VK_DESCRIPTOR_TYPE_COMBINED_IMAGE_SAMPLER, 1, VK_SHADER_STAGE_ALL, &sampler.handle()},
+                  });
+
+    if (descriptor_set.set_ == VK_NULL_HANDLE) {
+        GTEST_SKIP() << "Couldn't create descriptor set";
+    }
+
+    // 01564 appears to be impossible to hit due to the following check in descriptor_validation.cpp:
```
Comment (arno-lunarg) [inline tests/negative/ycbcr.cpp:1574]: should we open an issue regarding the problem described in this comment?

**Review by arno-lunarg — APPROVED** [review]
LGTM! 2 comments

File: tests/negative/ycbcr.cpp (hunk 1 above)
Comment (spencer-lunarg) [reply tests/negative/ycbcr.cpp:1536]: I just copied it over, but agree can fix this while here

File: tests/negative/ycbcr.cpp (hunk 2 above)
Comment (spencer-lunarg) [reply tests/negative/ycbcr.cpp:1574]: yes, I'll give it another 5 minutes to see if I can figure it out, if not, will add an issue number to it

**Review by ncesario-lunarg — APPROVED** [review]

File: tests/negative/ycbcr.cpp (hunk 1 above)
Comment (spencer-lunarg) [reply tests/negative/ycbcr.cpp:1536]: we are using `vk_testing::SamplerYcbcrConversion` so doing this showed me they were unused lol... even better

File: tests/negative/ycbcr.cpp (hunk 2 above)
Comment (spencer-lunarg) [reply tests/negative/ycbcr.cpp]: https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/5591

**Pull request merged**